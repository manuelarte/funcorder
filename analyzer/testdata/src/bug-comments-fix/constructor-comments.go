package bug_comments_fix

func NewConstructorComments() *ConstructorComments { // want `constructor "NewConstructorComments" for struct "ConstructorComments" should be placed after the struct declaration`
	// foo is foo
	return &ConstructorComments{Name: "John"}
}

type ConstructorComments struct {
	Name string
}
