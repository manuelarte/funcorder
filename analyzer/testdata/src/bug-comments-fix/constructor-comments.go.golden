package bug_comments_fix

type ConstructorComments struct {
	Name string
}

func NewConstructorComments() *ConstructorComments { // want `constructor "NewConstructorComments" for struct "ConstructorComments" should be placed after the struct declaration`
	// foo is foo
	return &ConstructorComments{Name: "John"}
}
